examples/website/texture-tester/components/textures-block.js

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import CompressedTexture from './texture';
+
+const Header = styled.h2`
```

Comment:
I appreciate the effort to make a componentized React app. But this component doesn't do a whole lot. Worth considering whether just to merge this into the app, it would just be a couple of lines added there.